modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/idp/mgt/ResidentIDPConfigsTestCase.java

```diff
@@ -40,230 +44,209 @@
+                        "Expected OAuth10a Request Token URL is not found"},
+                {"carbon.super", "IdentityProviderUrl", "https://localhost:9853/passivests",
+                        "Expected Passive STS URL is not found"},
+                {"carbon.super", "OpenIdUrl", "https://localhost:9853/openidserver",
+                        "Expected OpenID Server URL is not found"},
+                {"carbon.super", "ECPUrl", "https://localhost:9853/samlecp",
+                        "Expected ECP URL is not found"},
+                {"carbon.super", "LogoutReqUrl", "https://localhost:9853/samlsso",
+                        "Expected Logout URL is not found"},
+                {"carbon.super", "ArtifactResolveUrl", "https://localhost:9853/samlartresolve",
+                        "Expected Artifact Resolution URL is not found"},
+                {"carbon.super", "SSOUrl", "https://localhost:9853/samlsso",
+                        "Expected SSO URL is not found"},
+                {"carbon.super", "IDENTITY_PROVIDER_URL", "https://localhost:9853/services/wso2carbon-sts",
+                        "Expected Security Token Service URL is not found"},
+                {"carbon.super", "OIDCWebFingerEPUrl", "https://localhost:9853/.well-known/webfinger",
+                        "Expected Web finger Endpoint URL is not found"},
+                {"carbon.super", "IdPEntityId", "https://localhost:9853/oauth2/token",
+                        "Expected Token Endpoint URL is not found"},
+                {"carbon.super", "OIDCCheckSessionEPUrl", "https://localhost:9853/oidc/checksession",
+                        "Expected Session IFrame Endpoint URL is not found"},
+                {"carbon.super", "OAuth2IntrospectEPUrl", "https://localhost:9853/oauth2/introspect",
+                        "Expected Token Introspection Endpoint URL is not found"},
+                {"carbon.super", "OAuth2RevokeEPUrl", "https://localhost:9853/oauth2/revoke",
+                        "Expected Token Revocation Endpoint URL is not found"},
+                {"carbon.super", "OIDCLogoutEPUrl", "https://localhost:9853/oidc/logout",
+                        "Expected Logout Endpoint URL not found"},
+                {"carbon.super", "OAuth2AuthzEPUrl", "https://localhost:9853/oauth2/authorize",
+                        "Expected Authorization Endpoint URL is not found"},
+                {"carbon.super", "OAuth2DCREPUrl", "https://localhost:9853/api/identity/oauth2/dcr/v1.1/register",
+                        "Expected Dynamic Client Registration Endpoint URL is not found"},
+                {"carbon.super", "OIDCDiscoveryEPUrl", "https://localhost:9853/oauth2/oidcdiscovery",
+                        "Expected Dynamic Client Registration Endpoint URL is not found"},
+                {"carbon.super", "OAuth2JWKSPage", "https://localhost:9853/oauth2/jwks", "Expected is not found"},
```

Comment:
Shall we mention what is not found here as well?

Reply:
Fixed 40f5178